### DISCUSSION TITLE
issue in connecting my local environment to GCP environment.

### DISCUSSION THREAD

**Discussion by bharathravi-dev** [Sunbird-coKreat/Community#44] [General Discussions]
hi Team,
      i am facing a issue to start the server because it throwing me azure storage connection issue. i am using gcloud as my cloudprovider and i am using all keys of gcloud to connect it.

eventhough i am facing with the below error.

<img width="1792" alt="Screenshot 2022-12-30 at 12 15 39 PM" src="https://user-images.githubusercontent.com/117885618/210061308-d4ba716e-c74e-47a8-adcc-850db8218bb1.png">

i am using latest branch from github [release-5.1.0_RC7](for gcp env).

The issue is when i run my node server.

**Comment by arun1205** [comment]
@BharathTarento We would be fixing this on a priority basis

**Reply by arun1205** [reply]
This has been fixed and availaible in branch creation-portal/tree/release-5.1.0
